src/components/UserSearch.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
+
+class UserSearch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      githubUsername: '',
+      selectedUsername: '',
+    };
+
+    this.searchUser = this.searchUser.bind(this);
+    this.updateUsername = this.updateUsername.bind(this);
+  }
+
+  searchUser(evt) {
+    evt.preventDefault();
```

Comment:
What was your reasoning for`evt.preventDefault()` here?

Reply:
I didn't want to refresh the page from the form submission, but if I use the onClick you mentioned below then I won't have to do this. I figured since we are not dispatching an action for the search it should not refresh the page.